### ISSUE TITLE
Allow sorting of folders by date

### ISSUE THREAD

**Issue by tapesonthefloor** [owncloud/ios#318]
Pardon the potential dupe, but can folders/views be sorted by date as well as by name? I sync a huge notes folder, and only ever really need to access the one or two most recently modified. This'd be pretty consistent with expected smartphone behaviour, anyway, a la the "Camera Roll."

Thanks!

**Comment by nasli** [comment]
Hi @tapesonthefloor, thanks for your comment.
This feature is already registered in our product backlog, however there is not an eta yet.

**Comment by tapesonthefloor** [comment]
Thanks for the response, @nasli! Is that backlog open to public contributions?

**Comment by rperezb** [comment]
@tapesonthefloor of course contributors are more than welcome!
We do have some features labelled as junior job https://github.com/owncloud/ios/issues?q=is%3Aopen+is%3Aissue+label%3A%22Junior+Job%22 You may work on others, such as the one that you are asking about. This option is already done on the Android app, may be you want to have a look at: https://github.com/owncloud/android/pull/624

additionally, can you please have a look at https://github.com/owncloud/ios/blob/develop/CONTRIBUTING.md ?
Before starting contributing, we need you to sign the contributors agreement, https://owncloud.org/contribute/agreement/ and the iOS one https://owncloud.org/wp-content/themes/owncloudorgnew/assets/files/Addendum_iOS.pdf
thx!

**Comment by fasttech** [comment]
Yes pls.

**Comment by nasli** [comment]
You can keep up to date and suggest for the new feature in #391
